File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/models/PersonTest.java
package simulacionBandaMusica.models;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.factories.CantanteFactory;

import static org.junit.jupiter.api.Assertions.assertEquals;

class PersonTest {

    @Test
    void respirarTest() {
        Cantante cantanteTest = CantanteFactory.create();
        String correctString = "Estoy respirando...";
        assertEquals(correctString, cantanteTest.respirar());
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/models/PersonTest.kt
package simulacionBandaMusica.models

import org.junit.jupiter.api.Test
import simulacionBandaMusica.factories.CantanteFactory
import kotlin.test.assertEquals

internal class PersonTest {
    @Test
    fun respirarTest() {
        val cantanteTest: Musico.Cantante = CantanteFactory.create()
        val correctString = "Estoy respirando..."
        assertEquals(correctString, cantanteTest.respirar())
    }
}